### DISCUSSION TITLE
Please Update Voicemail

### DISCUSSION THREAD

**Discussion by batboyboy1990** [mikopbx/Core#650] [General ( Обо всём )]
Please Update Voicemail for this scrips :D!  i love this app

**Comment by boffart** [comment]
Do you need the functionality of classic voicemail?

**Reply by batboyboy1990** [reply]
Can you update voicemail on each number?? For example, 1001 calls for example *1 to listen to voicemail, 1002 is the same??

**Comment by andyjmorris** [comment]
How do you add your voicemail greeting?  How to dial into the voicemail system?  Or does MikoPBX only email out voicemails?  I do not record any calls so it doesn't take up too much hard drive space.  I really liked the old AskoziaPBX that did both email and you could dial into the system to check your voicemails. Kinda like the FreePBX to dial *97 or *98 to get to voicemail.
